UPGRADE.md

```diff
@@ -5,14 +5,27 @@ possible. Please note that not all breaking changes might be included here.
 Please check the [CHANGELOG.md](./CHANGELOG.md) for a full list of changes
 before finalizing the upgrade process.
 
-## unreleased
+## v0.8.0-alpha.1
```

Comment:
I suggest to simply delete this file! Make sure to remove it from any links as well.